deploy/README.md

```diff
@@ -5,3 +5,8 @@ Deployment configurations for additional services. Currently suitable for dev de
 ```shell
 docker compose up
 ```
+
+To run locally use ssh reverse port forwarding. For example:
```

Comment:
What is this for? I think the app will be available on localhost without SSH thanks to the port mapping in docker-compose.yml.

Reply:
for e2e testing via vk without white IP